DownloadTargetFiles.py

```diff
@@ -12,9 +12,10 @@
     ymd_tuple_to_mjd,
 )
 
+from pdart.pipeline.SuffixInfo import ACCEPTED_INSTRUMENTS  # type: ignore
+
 _YMD = Tuple[int, int, int]
 
-_ACCEPTED_INSTRUMENTS: str = "IJLNOSUVWXYZ"
 """
 We currently only handle products from a limited set of
 instruments.  These are the first letters of their 'obs_id's.
```

Comment:
Two lines down, I see a reference to "ACCEPTED_SUFFIXES". That seems like a leftover piece of code and should be removed or updated.